services/app-api/handlers/reports/fetch.ts

```diff
@@ -42,35 +42,36 @@ export const fetchReportsByState = handler(async (event, _context) => {
   };
 
   let startingKey;
-  let keepSearching = true;
   let existingItems = [];
   let results;
-  let count = 0;
 
-  const queryTable = async (keepSearching: boolean, startingKey?: any) => {
+  const queryTable = async (startingKey?: any) => {
     queryParams.ExclusiveStartKey = startingKey;
     let results = await dynamoDb.query(queryParams);
     if (results.LastEvaluatedKey) {
       startingKey = results.LastEvaluatedKey;
-      return [startingKey, keepSearching, results];
+      return [startingKey, results];
     } else {
-      keepSearching = false;
-      return [null, keepSearching, results];
+      return [null, results];
     }
   };
 
   // Looping to perform complete scan of tables due to 1 mb limit per iteration
-  while (keepSearching) {
-    count++;
-    if (count > 4) {
-      break;
-    }
-    [startingKey, keepSearching, results] = await queryTable(
-      keepSearching,
-      startingKey
-    );
-    existingItems.push(...results.Items);
-  }
+  do {
+    [startingKey, results] = await queryTable(startingKey);
+
+    /*
+     * Remove formTemplate and formData to get rid of excessive size that isn't needed
+     * on the dashboard when this call is used
+     */
+    const items: AnyObject[] = results.Items;
+    items.forEach((item: any) => {
+      delete item["formTemplate"];
+      delete item["formData"];
```

Comment:
Nit: Could these be accessed with dot notation instead? i.e. `item.formTemplate` and `item.formData`